### PR TITLE
Update index.md to include ERC-777

### PR DESCRIPTION
**Pull request by Rodney-O-C-Melby**
Added ERC-777 to token Standards, with a short description.

### CODE REVIEW

**Comment by wackerow** [comment]
@Rodney-O-C-Melby Thanks for the PR =).. not necessarily against this, but if we're going to add external links to this list, maybe we want to include other token standards as well, like 1155. @samajammin thoughts on adding 777 to this list?

**Comment by samajammin** [comment]
Thanks for the PR! I think ideally we build out pages to explain each token but in the meantime I'm perfectly happy to add links to popular token standards.

File: src/content/developers/docs/standards/tokens/index.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ Here are some of the most popular token standards on Ethereum:
 
 - [ERC-20](/developers/docs/standards/tokens/erc-20/) - A standard interface for fungible (interchangeable) tokens, like voting tokens, staking tokens or virtual currencies.
 - [ERC-721](/developers/docs/standards/tokens/erc-721/) - A standard interface for non-fungible tokens, like a deed for artwork or a song.
+- [ERC-777](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-777.md) - ERC-777 allows people to build extra functionality on top of tokens such as a mixer contract for improved transaction privacy or an emergency recover function to bail you out if you lose your private keys.
```
Comment (samajammin) [inline src/content/developers/docs/standards/tokens/index.md:24]:
```suggestion
- [ERC-777](https://eips.ethereum.org/EIPS/eip-777) - ERC-777 allows people to build extra functionality on top of tokens such as a mixer contract for improved transaction privacy or an emergency recover function to bail you out if you lose your private keys.
```

Comment (samajammin) [reply src/content/developers/docs/standards/tokens/index.md:24]: Subjective opinion but I think linking to the EIP site is better UX.

Comment (wackerow) [reply src/content/developers/docs/standards/tokens/index.md:24]: Agree here, plus this link contains the GitHub link right at the top.

**Review by wackerow — APPROVED** [review]
@Rodney-O-C-Melby Made the small change Sam suggested (see above). Bringing in... Thanks again! 👍🏻

**Comment by wackerow** [comment]
@all-contributors please add @Rodney-O-C-Melby for content addition

**Pull request merged**